superset-frontend/cypress-base/cypress/e2e/dashboard/drillby.test.ts

```diff
@@ -0,0 +1,712 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Interception } from 'cypress/types/net-stubbing';
+import { waitForChartLoad } from 'cypress/utils';
+import { SUPPORTED_CHARTS_DASHBOARD } from 'cypress/utils/urls';
+import { SUPPORTED_TIER1_CHARTS, SUPPORTED_TIER2_CHARTS } from './utils';
+
+const interceptV1ChartData = (alias = 'v1Data') => {
+  cy.intercept('/api/v1/chart/data*').as(alias);
+};
+
+const interceptLegacyChartData = () => {
+  cy.intercept('/superset/explore_json*').as('legacyData');
+};
+
+const interceptFormDataKey = () => {
+  cy.intercept('/api/v1/explore/form_data').as('formDataKey');
+};
+
+const closeModal = () => {
+  cy.get('body').then($body => {
+    if ($body.find('[data-test="close-drill-by-modal"]').length) {
+      cy.getBySel('close-drill-by-modal').click({ force: true });
+    }
+  });
+};
+
+const setTopLevelTab = (tabName: string) => {
```

Comment:
I think we have this already in drilltodetail tests. Should it be moved to the utils?